docs/release-notes/README.md

```diff
@@ -0,0 +1,43 @@
+# Release Notes
+
+We want release notes that are accurate, comprehensive, and written so that
```

Comment:
missing a "to"

Reply:
🤔 hmm, where did you want the "to" added?